src/main/src/test/java/org/geoserver/test/GeoServerBaseTestSupport.java

```diff
@@ -136,7 +139,15 @@ public final void doSetup() throws Exception {
         if (testData == null) {
             test = this;
             testData = createTestData();
-            testData.setUp();
+
+            Logger imageLogger = Logger.getLogger("org.geotools.image");
```

Comment:
Interesting, why the new addition? Isn't this out of line with the ability to choose a logging profile while writing tests?
I often override getLogConfiguration, when a test is failing, to get more information. But this one bit is static, would not follow the chosen logging profile.

Reply:
If I remember it is due to an oddity of the java until logging which can be used when running some unit tests; if a logger is not instantiated, then the level is not available for child loggers.

Reply:
Would it allow customization of the logging level of "org.geotools.image" by overriding ``GeoSeverSystemTestSupport.getLogConfiguration``, or not?

Reply:
sorry looking at the complete code example, this was to suppress endless info messages during setup before restoring the prior level.

Reply:
So If I understand your feedback you would prefer changing TEST_LOGGING.xml to reduce the level for `org.geotools.image`; trouble is I only want to reduce the level during setup....